modules/030-cloud-provider-yandex/openapi/config-values.yaml

```diff
@@ -1,5 +1,11 @@
 type: object
 properties:
+  additionalExternalNetworkIDs:
+    type: array
+    items:
+      type: string
+    default: []
+    description: A list of Network IDs that will be considered `ExternalIP` when listing Node addresses. An optional parameter
```

Comment:
You missed a dot after the end of the description.

Reply:
fixed